Tasks/MSBuild/Helpers.ps1

```diff
@@ -60,6 +60,11 @@ function Invoke-BuildTools {
         [switch]$NoTimelineLogger
     )
 
+    if ($NuGetRestore)
+    {
+        Write-Warning (Get-LocalizedString -Key "The `"Restore NuGet Packages`" option is deprecated. To restore NuGet packages in your build, add a NuGet Installer task to your build definition.")
```

Comment:
@ericsciple it would be awesome to simply not have the checkbox in the new task, and set up the default Visual Studio Build template to include a NuGet Installer step. Is that possible?